frontend/packages/ux-editor/src/components/config/editModal/EditImage/LocalImage/ImportImage/AddImageFromLibrary/ChooseFromLibrary/ChooseFromLibrary.module.css

```diff
@@ -0,0 +1,32 @@
+.cardsContainer {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  --gap-cards: var(--fds-spacing-2);
+  gap: var(--gap-cards);
+  --padding-cards: var(--fds-spacing-2);
+  padding: var(--padding-cards);
+}
+
+.card {
+  cursor: pointer;
+}
+
+.card img {
+  width: 100%;
+  height: 150px;
+  margin: 0 auto;
+  object-fit: cover;
+}
+
+.fileName {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  max-width: calc(
+    (var(--studio-file-changes-modal-width) / 3) - 3 * var(--gap-cards) - 4 * var(--padding-cards)
+  );
```

Comment:
Hvor kommer tallene 3 og 4 fra i denne utregningen? Kanskje du kan lage egne variabler for dem også?